cmd/juju/controller/destroy.go

```diff
@@ -119,6 +121,16 @@ WARNING! This command will destroy the %q controller.
 This includes all machines, applications, data and other resources.
 `[1:]
 
+var destroySysMsgWithDetails = `
+WARNING! This command will destroy the %q controller.
+This includes all data and other resources:
+ - %d model(s) will be destroyed
+  - model list: %q
```

Comment:
This won't look great if there are no models

Reply:
If there is no model, we will use another string template:

```
var destroySysMsg = `
WARNING! This command will destroy the %q controller.
This includes all machines, applications, data and other resources.
`[1:]
```

you can check it with empty model:
```
juju bootstrap lxd lxd
juju destroy-controller lxd
```